packages/platforms/accton/x86-64/as5915-18x/onlp/builds/x86_64_accton_as5915_18x/module/inc/x86_64_accton_as5915_18x/x86_64_accton_as5915_18x.x

```diff
@@ -0,0 +1,14 @@
+/**************************************************************************//**
+ *
+ *
+ *
+ *****************************************************************************/
+#include <x86_64_accton_as5915_18x/x86_64_accton_as5915_18x_config.h>
+
+/* <--auto.start.xmacro(ALL).define> */
+/* <auto.end.xmacro(ALL).define> */
+
+/* <--auto.start.xenum(ALL).define> */
+/* <auto.end.xenum(ALL).define> */
+
+
```

Comment:
Please remove blank lines at the end of a file.

Reply:
Removed.